Pegasus/include/Geometry.hpp

```diff
@@ -165,18 +165,29 @@ class Box : public SimpleShape
 namespace intersection
 {
 // Utility functions
+
+/**
+ * @brief Calculates box vertices in the model coordinate space from a given orthogonal basis
+ * @tparam Random access iterator
+ * @param[in] i box axis vector
+ * @param[in] j box axis vector
+ * @param[in] k box axis vector
+ * @param[out] verticesIterator iterator to the container that is able to store 8 vertices
```

Comment:
But the supplied variable is not changed, it still points to the same element it did before the method call.
It's like when `const` is applied to a method - `this` is const, but pointer members aren't (`A* a;` -> `A* const a;`).